index.html

```diff
@@ -0,0 +1,754 @@
+                    </li>
+                    <li><a href="#" class="testing">Google Maps can now highlight wheelchair-accesible busnesses</a>
+                    </li>
+                    <li><a href="#" class="testing">Facebook announces Workplace Rooms, basically Messenger Rooms for
+                            work</a>
+                    </li>
+                    <li><a href="#" class="testing">TikTok now gives you one-tap access to streaming app Resso</a></li>
+                    <li><a href="#" class="testing">Alook at how Jitsi became a 'secure' open-source alternative to
+                            Zoom</a>
+                    </li>
+                </ul>
+
+            </section>
+            <section class="popular">
+                <h2>Distract ></h2>
+                <ul class="listOne">
+                    <li>
+                        <div class="sectionPopular8">
+                            <a href="#">
+                                <div class="news__title">
+                                    <h5 class="white no-margin mostPopular">Most popular</h5>
+                                    <h3 class="white no-margin">Chrome is testing dark mode for Google Search</h3>
+                                    <div class="news__author no-margin padding-top">
+                                        Mix 5 hours ago
+                                    </div>
+                                </div>
+                            </a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="latest">
+                            <div class="laTitles">
+                                <h5 class="no-margin">LATEST</h5>
+                                <h4 class="no-margin">Icon Rewind is a time machine for your phone’s app icons </h4>
+                            </div>
+                            <img src="assets/images/icon3.png" alt="icon 1">
+                        </div>
+                    </li>
+                </ul>
+                <ul class="listTwo">
+                    <li><a href="#" class="testing">YouTube now tells you to go to bed -- here's how to set it up</a>
+                    </li>
+                    <li><a href="#" class="testing">Google Maps can now highlight wheelchair-accesible busnesses</a>
+                    </li>
+                    <li><a href="#" class="testing">Facebook announces Workplace Rooms, basically Messenger Rooms for
+                            work</a>
+                    </li>
+                    <li><a href="#" class="testing">TikTok now gives you one-tap access to streaming app Resso</a></li>
+                    <li><a href="#" class="testing">Alook at how Jitsi became a 'secure' open-source alternative to
+                            Zoom</a>
+                    </li>
+                </ul>
+
+            </section>
+        </div>
+
+        <div class="latestNews">
```

Comment:
In the "Latest Deals" section, each row should have 3 boxes in medium screen mode. Right now it continues with the styles for the desktop screens.

Reply:
Thanks for your review Tirthajyoti, it has been really helpful. All the changes you requested were made!